### DISCUSSION TITLE
Better Subscription Options

### DISCUSSION THREAD

**Discussion by JamesMcGhee** [GitbookIO/community#97] [Feature Requests]
### What's your feature/idea?

Give me a reason to subscribe to your product ... and a subscription plan that makes since for me to use.

1st I think my use case is not one you thought of so you dont really have a means for me to monetize for you that makes since for me to do. So here is a bit about me and why I think more like me would love to use your tool and pay you for it.

Before I write a wall of text here is the TL;DR
Plus is useless to us, Pro at 114 a month make ZERO since for us thus we remain on free 

So who are we and why do you care?
We are a tiny little dev house creating games and game software tools like our Steam integration.
We use GitBook to write our KB which is a huge value to us and our community and we use GitBook to write our design documents.

So in bucket 1 our KB ... that is much more like exactly what you probably envisioned GitBook to be used for ... we are WAY to small to need any of the tech features in Pro e.g. Team Management, Visitor Auth, etc. .... but it is a big part of our brand so we would LOVE to brand it better ... e.g. better customization.
We would really like to pay you money to have access to better customization options ...but no we will not be paying 114 a month so we can prety-fy our KB its not horrible as it is the "Plus" option is useless for us here and offers us nothing of value we don't already have in free.

By the way we are not the only asset / tool dev in the game space that uses your tool to auth a KB, we have seen others use you and clearly they are the free version as well. Your missing a market there of the small time software dev that has ZERO use for login or team management BS but runs a little solo biz and so REALLY wants to brand that KB they are putting huge time into.

So in bucket 2
As a game dev ... we love using GetBook its supper easy to write in, easy to keep formating nice, easy to back up and easy to share ... thoguh ... you know "unlisted" was a free publsih option and you made it a Pro only option now ... slightly dick move. We really have a need for unlisted but again 114 a month ... NOPE.

We again dont need team management, we are a VERY small team at most we might find it handy to have multiple people who can edit. Branding is less important in the game dev side but would be a nice to have ... publishing unlisted though ... that one is worth a few bob a month ... not 114 a month but a few bob.

In short if you had a subscription plan that was a few quid a month, could have as few as 1 person on it and covered branding type things (better customization) and unlisted publish option ... we would sign right up. 

As it is Plus gives us a few things but we have ZERO use for any of them so that is useless and Pro which has a couple things we would like and use is 114 a month are you crazy we are not an enterprise, we will not have a team of 100 people working on the docs that will never make any since for us and I do mean never.

### What's your desired outcome?

We would love to see a subscription option that was aimed at us

a few bucks a month all we need that isn't in free is customization options for better branding our KB and unlisted publish option for our design docs.

And yes we do publish our design docs knowing the world can see them ... we plan on converting them to online game manuals (public listing) later ... and again branding would be important in that form

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by aikoii** [comment]
Hi @JamesMcGhee!

Thank you for sharing your feedback. Our paid plans have features that are resource-heavy, and those fees help us scale this. We have tried to make sure that the vast majority of our users have a pricing plan that works for them, I'm sorry to hear this is not the case for you.

I understand this is not the answer that you were hoping for right now. I hope that we'll be able to better accommodate your needs in the future 🙂.

**Reply by JamesMcGhee** [reply]
Thanks, would love to pay you, but so far, there isn't anything for me to buy that I can justify. 

Hope to see that change some day.
